terraform/dev/terragrunt.hcl

```diff
@@ -81,6 +81,28 @@ locals {
   LAMBDA_LAYER_KEY = get_env("LAMBDA_LAYER_KEY")
   FUNCTION_BUCKET = get_env("FUNCTION_BUCKET")
   WFONE_NOTIFICATIONS_API_IMAGE = get_env("WFONE_NOTIFICATIONS_API_IMAGE")
+  
+  # WFONE_NOTIFICATIONS_API
+  WFONE_NOTIFICATIONS_API_DATASOURCE_MAX_CONNECTIONS = get_env("WFONE_NOTIFICATIONS_API_DATASOURCE_MAX_CONNECTIONS")
+  WFONE_NOTIFICATIONS_API_DATASOURCE_URL = get_env("WFONE_NOTIFICATIONS_API_DATASOURCE_URL")
+  WFONE_NOTIFICATIONS_API_DATASOURCE_USER = get_env("WFONE_NOTIFICATIONS_API_DATASOURCE_USER")
+  WFONE_NOTIFICATIONS_API_DATASOURCE_PASSWORD = get_env("WFONE_NOTIFICATIONS_API_DATASOURCE_PASSWORD")
+  WFONE_NOTIFICATIONS_API_EMAIL_ADMIN_EMAIL = get_env("WFONE_NOTIFICATIONS_API_EMAIL_ADMIN_EMAIL")
+  WFONE_NOTIFICATIONS_API_EMAIL_FROM_EMAIL = get_env("WFONE_NOTIFICATIONS_API_EMAIL_FROM_EMAIL")
+  WFONE_NOTIFICATIONS_API_EMAIL_NOTIFICATIONS_ENABLED = get_env("WFONE_NOTIFICATIONS_API_EMAIL_NOTIFICATIONS_ENABLED")
+  WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SEND_ERROR_FREQ = get_env("WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SEND_ERROR_FREQ")
+  WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SEND_ERROR_SUBJECT = get_env("WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SEND_ERROR_SUBJECT")
+  WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SEND_FREQ = get_env("WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SEND_FREQ")
+  WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SUBJECT = get_env("WFONE_NOTIFICATIONS_API_EMAIL_SYNC_SUBJECT")
+  WFONE_NOTIFICATIONS_API_PUSH_ITEM_EXPIRE_HOURS = get_env("WFONE_NOTIFICATIONS_API_PUSH_ITEM_EXPIRE_HOURS")
+  WFONE_NOTIFICATIONS_API_QUARTZ_CONSUMER_INTERVAL_SECONDS = get_env("WFONE_NOTIFICATIONS_API_QUARTZ_CONSUMER_INTERVAL_SECONDS")
+  WFONE_NOTIFICATIONS_API_SMTP_CREDENTIALS_PASSWORD = get_env("WFONE_NOTIFICATIONS_API_SMTP_CREDENTIALS_PASSWORD")
+  WFONE_NOTIFICATIONS_API_SMTP_CREDENTIALS_USER = get_env("WFONE_NOTIFICATIONS_API_SMTP_CREDENTIALS_USER")
+  WFONE_NOTIFICATIONS_API_SMTP_HOST_NAME = get_env("WFONE_NOTIFICATIONS_API_SMTP_HOST_NAME")
+  WFONE_NOTIFICATIONS_API_WEBADE_OAUTH2_CLIENT_ID = get_env("WFONE_NOTIFICATIONS_API_WEBADE_OAUTH2_CLIENT_ID")
+  WFONE_NOTIFICATIONS_API_WEBADE_OAUTH2_REST_CLIENT_SECRET = get_env("WFONE_NOTIFICATIONS_API_WEBADE_OAUTH2_REST_CLIENT_SECRET")
+  WFONE_NOTIFICATIONS_API_WEBADE_OAUTH2_WFIM_CLIENT_ID = get_env("WFONE_NOTIFICATIONS_API_WEBADE_OAUTH2_WFIM_CLIENT_ID")
+  WFSS_POINTID_URL = get_env("WFSS_POINTID_URL")
```

Comment:
This one can be removed, since it will be generated in this project - rather than passing it in as a variable, you should be able to use aws_cloudfront_distribution.wfss_pointid_api.aliases[0]